src/ansys/dpf/core/examples/downloads.py

```diff
@@ -1993,3 +2000,142 @@ def find_distributed_msup_folder(
+        "average_filter_plugin/common.py",
+    ]
+    return _retrieve_plugin(
+        file_list=file_list,
+        should_upload=should_upload,
+        server=server,
+        return_local_path=return_local_path,
+    )
+
+
+def download_gltf_plugin(
+    should_upload: bool = True, server=None, return_local_path=False
+) -> Union[str, None]:
+    """Make the plugin available server side, if the server is remote the plugin is uploaded
+    server side. Returns the path of the plugin folder.
+
+    Parameters
+    ----------
+    should_upload:
+        Whether the file should be uploaded server side when the server is remote.
+    server:
+        Server with channel connected to the remote or local instance. When
+        ``None``, attempts to use the global server.
+    return_local_path:
+        If ``True``, the local path is returned as is, without uploading, nor searching
+        for mounted volumes.
+
+    Returns
+    -------
+    str
+        Path to the plugin folder.
+
+    Examples
+    --------
+
+    >>> from ansys.dpf.core import examples
+    >>> path = examples.download_gltf_plugin()
+
+    """
+    file_list = [
+        "gltf_plugin.xml",
+        "gltf_plugin/__init__.py",
+        "gltf_plugin/operators.py",
+        "gltf_plugin/operators_loader.py",
+        "gltf_plugin/requirements.txt",
+        "gltf_plugin/gltf_export.py",
+        "gltf_plugin/texture.png",
+    ]
+    return _retrieve_plugin(
+        file_list=file_list,
+        should_upload=should_upload,
+        server=server,
+        return_local_path=return_local_path,
+    )
+
+
+def download_easy_statistics(
+    should_upload: bool = True, server=None, return_local_path=False
+) -> Union[str, None]:
+    """Make the plugin available server side, if the server is remote the plugin is uploaded
+    server side. Returns the path of the plugin folder.
+
+    Parameters
+    ----------
+    should_upload:
+        Whether the file should be uploaded server side when the server is remote.
+    server:
+        Server with channel connected to the remote or local instance. When
+        ``None``, attempts to use the global server.
+    return_local_path:
+        If ``True``, the local path is returned as is, without uploading, nor searching
+        for mounted volumes.
+
+    Returns
+    -------
+    str
+        Path to the plugin folder.
+
+    Examples
+    --------
+
+    >>> from ansys.dpf.core import examples
+    >>> path = examples.download_easy_statistics()
+
+    """
+    file_name = "easy_statistics.py"
+    EXAMPLE_FILE = GITHUB_SOURCE_URL + file_name
+    operator_file_path = _retrieve_file(
+        EXAMPLE_FILE, filename=file_name, directory="python_plugins"
+    )
+    return find_files(operator_file_path, should_upload, server, return_local_path)
+
+
+def _retrieve_plugin(
+    file_list: list[str], should_upload: bool = True, server=None, return_local_path=False
+) -> Union[str, None]:
```

Comment:
This can also be seen as a `Optional[str]`
But in the same time python3.10 expects us to write it as a `str | None` so the union might be more future-proof

Reply:
Until we drop support for 3.9 completely I tend to use `Union`.